### PR TITLE
Add missing examples in SocketAddr

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
r? @frewsxcv

### CODE REVIEW

File: src/libstd/net/addr.rs [hunk 1]
```diff
@@ -106,6 +168,20 @@ impl SocketAddr {
...
     /// false if it's a valid IPv4 address.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// #![feature(sockaddr_checker)]
+    ///
+    /// use std::net::{IpAddr, Ipv6Addr, SocketAddr};
+    ///
+    /// fn main() {
```
Comment (frewsxcv) [inline src/libstd/net/addr.rs]: Needs a closing brace.

**Comment by GuillaumeGomez** [comment]
Updated.

**Review by frewsxcv — APPROVED** [review]

**Pull request merged**